server/app/services/migration/ProgramMigrationService.java

```diff
@@ -146,6 +147,9 @@ public ProgramDefinition prepForExport(ProgramDefinition programDefinition) {
         .setAcls(new ProgramAcls())
         // Don't export environment specific notification preferences
         .setNotificationPreferences(ImmutableList.of())
+        // Explicitly set program type to DEFAULT so we don't import program as a
+        // pre-screener/common intake
+        .setProgramType(ProgramType.DEFAULT)
```

Comment:
**note:** I didn't know the `ProgramDefinition` had a `ProgramType` enum.  Very handy!

Reply:
right?? made this a super easy fix.